src/features/nft/addNFT/fullScreenButton/FullScreenButton.tsx

```diff
@@ -1,16 +1,26 @@
 import React from 'react'
+import cn from 'classnames'
 import { Expand } from 'common/components/Icons'
-import style from './FullScreenButton.module.css'
 
 type TProps = {
   onClick(): void
+  disabled?: boolean
 }
 
-export default function FullScreenButton({ onClick }: TProps): JSX.Element {
+export default function FullScreenButton({
```

Comment:
I think it's better to use arrow function for keeping the same style throughout the project.
const FullScreenButton = () => {}

Reply:
It's more compact to use function with default export
And with `const` you need to scroll down the file to see if it is default exported

```js
const FullScreenButton = () => {}

export default FullScreenButton
```
vs
```js
export default function FullScreenButton() {}
```

Reply:
agreed with @CrazyOptimist